training/heterogeneous-clusters/hello.world.sagemaker/start_job.py

```diff
@@ -0,0 +1,32 @@
+import datetime
+from sagemaker.tensorflow import TensorFlow
+from sagemaker.instance_group import InstanceGroup
+import os
+
+REGION = 'us-east-1'
```

Comment:
We've recently started doing multi-region testing, so I imagine this is going to break. Does this solution only work in us-east-1?

Reply:
I'm removing all references to us-east-1. The notebooks should work in all regions that with the instance types we test/benchmark with: ml.g5.*, and ml.p4d.24xlarge.
Heterogeneous clusters feature specifically focuses and improve highend compute instances with fixed amount of CPU:GPU, therefore we benchmark with a P4d which is the best candidate for users of the feature, however, a user could change to other instance types as they see fit.